text/0025-core-compute-resources-as-plugins.md

```diff
@@ -0,0 +1,77 @@
+- Feature Name: Core Compute Resources as Plugins
+- Start Date: 2016-10-21
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Core Compute Resources should move to plugins
+
+# Motivation
+[motivation]: #motivation
+
+Core has been set up to allow compute resources as plugins for some while, and
+we already have several examples of this in use (Xen, Azure, DigitalOcean,
+etc). This has brought benefits:
+
+* able to release independatly of Core
```

Comment:
while its true, many of the dependencies go back to fog, so unless fog is broken down fully to different providers, one plugin requirements might conflict with another plugin, making the support matrix harder to follow